### ISSUE TITLE
Python libraries have invalid signature on macOS/arm64

### ISSUE THREAD

**Issue by fxcoudert** [conda-forge/python-feedstock#563]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

I updated with `conda update`:

```
The following packages will be UPDATED:

  itsdangerous                           2.1.1-pyhd8ed1ab_0 --> 2.1.2-pyhd8ed1ab_0
  openssl                                 1.1.1l-h3422bc3_0 --> 1.1.1n-h90dfc92_0
  python                          3.9.10-hd16f9c5_2_cpython --> 3.9.12-hfc7342c_0_cpython
  tqdm                                  4.63.0-pyhd8ed1ab_0 --> 4.63.1-pyhd8ed1ab_0
```

and now python cannot run anymore because its libraries all have invalid signatures:

```
$ python3.9     
zsh: killed     python3.9
$ codesign -v -vvv =python3.9
/Users/fx/conda3/bin/python3.9: valid on disk
/Users/fx/conda3/bin/python3.9: satisfies its Designated Requirement
$ codesign -v -vvv /Users/fx/conda3/lib/python3.9/lib-dynload/zlib.cpython-39-darwin.so
/Users/fx/conda3/lib/python3.9/lib-dynload/zlib.cpython-39-darwin.so: invalid signature (code or signature have been modified)
In architecture: arm64
```

All libs in `conda3/lib/python3.9/lib-dynload/` have the same problem. Of course, since python cannot run, I can't update again or downgrade, or do anything really.

### Installed packages

```shell
I cannot run `conda list` because python does not run.
```

### Environment info

```shell
I cannot run `conda info` because python does not run. It's a macOS 12.3 on Apple Silicon (arm64).
```

**Comment by isuruf** [comment]
Fixed in https://github.com/conda-forge/python-feedstock/pull/564

**Comment by isuruf** [comment]
Fixed now. Thanks for the report.

**Comment by fxcoudert** [comment]
Confirmed fixed with `3.9.12-hfc7342c_1_cpython`. Thanks!

**Comment by beckermr** [comment]
closing this since it appears to be resolved. feel free to reopen if we find other issues
